db/seeds.rb

```diff
@@ -5,3 +5,47 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'nokogiri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.